backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/MaskinportenSchemaControllerTest.cs

```diff
@@ -153,17 +154,58 @@ public async Task PostMaskinportenSchemaDelegation_DAGL_Success()
         /// Expected: RevokeOfferedMaskinportenScopeDelegation returns 204 No Content
         /// </summary>
         [Fact]
-        public async Task RevokeOfferedMaskinportenScopeDelegation_Success()
+        public void RevokeOfferedMaskinportenScopeDelegation_Success()
```

Comment:
woopsie: Didn't actually mean to check in this change to the test as it was to try and reproduce the issue with multiple requests. But it fails to reproduce the issue. Will rollback the test change.

or could just add it as it's own test as it's still a valid test even if it's multiples of the same revoke